source/uwp/Renderer/lib/XamlBuilder.h

```diff
@@ -172,7 +181,14 @@ AdaptiveNamespaceStart
             _In_ bool insertSeparator,
             _Inout_ AdaptiveNamespace::AdaptiveRenderContext* renderContext,
             ABI::AdaptiveNamespace::ContainerStyle containerStyle);
-
+        
+        // I think this class is not used anymore, I think they changed the name for separator
+        /*
+        void GetSeparationConfigForElement(
+            _In_ ABI::AdaptiveCards::XamlCardRenderer::IAdaptiveCardElement* element,
+            _In_ ABI::AdaptiveCards::XamlCardRenderer::SeparationStyle separation,
+            _COM_Outptr_result_maybenull_ ABI::AdaptiveCards::XamlCardRenderer::IAdaptiveSeparationConfig** separationConfig);
+            */
```

Comment:
This is also an old version of an existing file (see line 220) and can be removed.